CHANGELOG.md

```diff
@@ -4,6 +4,61 @@
 	## __WORK IN PROGRESS__
 -->
 
+## __WORK IN PROGRESS__ - Kiera
+
+**Breaking changes**
+* Support for Node.JS 16 is dropped!
+* Binary states have been removed
+* Let's encrypt has been removed
+
+**Features**
+* (foxriver76) Added possibility to automatically upgrade adapters (see https://github.com/ioBroker/ioBroker.js-controller?tab=readme-ov-file#automatic-adapter-upgrade)
+* (foxriver76) if updates for OS packages are available a notification is generated (Linux only)
+* (foxriver76) js-controller (and thus the whole ioBroker) is now running as an ESM module internally while staying a 100 % backward compatible to adapters written in cjs
+* (foxriver76) the controller creates a notification if free disk space is critical (see https://github.com/ioBroker/ioBroker.js-controller?tab=readme-ov-file#disk-space-warnings)
+* (foxriver76) allow to ignore specific adapter versions (see https://github.com/ioBroker/ioBroker.js-controller?tab=readme-ov-file#ignoring-specific-adapter-version)
+* (foxriver76) generate notification if an adapter is blocklisted and thus stopped 
```

Comment:
i would turn that around because foirst of all we stop adapters nochactively .. and generate notifies, right?